sync2jira/upstream_issue.py

```diff
@@ -257,26 +311,37 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
-        if not issue.get('storypoints', None):
-            issue['storypoints'] = ''
+        issue_updates = config['sync2jira']['map']['github'][upstream].get('issue_updates', {})
+        if 'github_project_fields' in issue_updates:
+            issue['storypoints'] = None
+            issue['priority'] = ''
```

Comment:
Is there unit test coverage for this code? (I.e., lines 314-344)

Reply:
added test cases.